docs/src/code-examples/sending-emails/go.mdx

```diff
@@ -12,7 +12,7 @@ options := []notify.SendEmailOption{
 resp, err := client.SendEmail(templateID, emailAddress, options...)
 ```
 
-<Panel label={<H3>Arguments</H3>}>
```

Comment:
@andrewoh531 as discussed, probably better to keep the semantic h3, but just render as plain.